src/components/PortfolioProject/PortfolioProjectDesktop.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { FC } from 'react';
+
+import { PortfolioProjectDialogProps } from './PortfolioProject';
+import { IconButton } from '../IconButton/IconButton';
+import { Tag } from '../Tag/Tag';
+import { Button } from '../Button/Button';
+
+import { X, ChevronLeft, ChevronRight, Code, Image, ExternalLink } from 'react-feather';
+import { Box, Dialog, makeStyles, Typography } from '@material-ui/core/';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    '& .MuiDialog-paper': {
+      overflowY: 'inherit',
+    },
+    '& .MuiBackdrop-root': {
+      backgroundColor: fade(theme.palette.text.primary, 0.8),
+    },
+    '& .MuiPaper-rounded': { borderRadius: 15 },
+    overflowY: 'auto',
+    '&::-webkit-scrollbar': {
+      width: '0.5em',
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: fade(theme.palette.text.primary, 0.8),
+    },
+    '& .MuiDialog-container': {
+      height: 'auto',
+    },
+  },
+  iconButtons: {
+    position: 'absolute',
+    left: '103%',
+  },
+  tags: {
+    display: 'flex',
+    justifyContent: 'start',
+    flexWrap: 'wrap',
+    padding: theme.spacing(0, 3, 3, 3),
+    '& > *': {
+      margin: theme.spacing(0.5, 1, 0.5, 0),
+    },
+  },
+  iconButtonX: {
+    margin: theme.spacing(2, 0, 5, 0),
+  },
+  iconButtonChevronLeft: {
```

Comment:
You can pass `mb={2.5}` directly to the Box component so you don't need this class